bases/br_inep_censo_escolar/matricula/publish.sql

```diff
@@ -0,0 +1,91 @@
+/*
+
+Query para publicar a tabela.
+
+Esse é o lugar para:
+    - modificar nomes, ordem e tipos de colunas
+    - dar join com outras tabelas
+    - criar colunas extras (e.g. logs, proporções, etc.)
+
+Qualquer coluna definida aqui deve também existir em `table_config.yaml`.
+
+# Além disso, sinta-se à vontade para alterar alguns nomes obscuros
+# para algo um pouco mais explícito.
+
+TIPOS:
+    - Para modificar tipos de colunas, basta substituir STRING por outro tipo válido.
+    - Exemplo: `SAFE_CAST(column_name AS NUMERIC) column_name`
+    - Mais detalhes: https://cloud.google.com/bigquery/docs/reference/standard-sql/data-types
+
+*/
+
+CREATE VIEW basedosdados-dev.br_inep_censo_escolar.matricula AS
+SELECT 
+SAFE_CAST(ano AS INT64) ano,
+SAFE_CAST(sigla_uf AS STRING) sigla_uf,
+SAFE_CAST(id_municipio AS INT64) id_municipio,
+SAFE_CAST(rede AS STRING) rede,

```

Comment:
Outro detalhe é a ordem das colunas, a variável `rede` tá em posições diferente entre as tabelas

Reply:
já refiz tudo la em dev diego! to só esperando o @rdahis me responder sobre as descrições que faço o PR

Reply:
não precisa esperar o @rdahis pra fz commit.